docs/madr/1-intial.md

```diff
@@ -0,0 +1,155 @@
+# An event driven ANT registration system in AO to optimize ANT listings by access role
```

Comment:
You're suggesting the implementation approach in this title.

```suggestion
# ArNS ANT Registry
```

Reply:
https://github.com/ar-io/ar-io-ant-registry-process/pull/1#discussion_r1689177601